zerver/tests/test_new_users.py

```diff
@@ -145,17 +145,19 @@ def setUp(self) -> None:
              'Chrome/56.0.2924.87 Electron/1.6.8 Safari/537.36', 'Zulip', 'Windows'),
             ('Mozilla/5.0 (X11; Linux i686) AppleWebKit/535.7 (KHTML, '
              'like Gecko) Ubuntu/11.10 Chromium/16.0.912.77 '
-             'Chrome/16.0.912.77 Safari/535.7', 'Chromium', 'Linux'),
+             'Chrome/16.0.912.77 Safari/535.7', 'Chromium', 'Ubuntu'),
             ('Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 '
              '(KHTML, like Gecko) Chrome/28.0.1500.52 Safari/537.36 '
              'OPR/15.0.1147.100', 'Opera', 'Windows'),
-            ('Mozilla/5.0 (Windows NT 10.0; <64-bit tags>) AppleWebKit/'
-             '<WebKit Rev> (KHTML, like Gecko) Chrome/<Chrome Rev> Safari'
-             '/<WebKit Rev> Edge/<EdgeHTML Rev>.'
-             '<Windows Build>', 'Edge', 'Windows'),
+            ('Mozilla/5.0 (Windows NT 10.0; Win64; x64; WebView/3.0) ' +
+             'AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 ' +
+             'Safari/537.36 Edge/18.17763', 'Edge', 'Windows'),
```

Comment:
Corrected this as user-agent string earlier was not an actual user-agent, it was a format of user-agent of Edge browser in Windows.